core/src/main/java/se2/ticktackbumm/core/client/ClientMessageHandler.java

```diff
@@ -109,4 +109,9 @@ public void handleGameFinished(GameFinished gameFinished) {
     public void handleStartBomb(BombStart bombStart) {
         game.setBombToTick(bombStart.getBombTimer());
     }
+
+    public void handleSpinWheelFinished(SpinWheelFinished spinWheelFinished){
+        Log.info(LOG_TAG, "<SpinWheelFinished-CurrentGameMode> : "  + spinWheelFinished.getGameMode());
+        game.spinWheelFinished(spinWheelFinished.getGameMode());
```

Comment:
Hier muss der GameMode nicht übergeben werden, da er bereits mit GameUpdate in der GameData des Clients aktualisiert wurde.